src/scripts.js

```diff
@@ -1 +1,140 @@
-console.log("Hello World");
\ No newline at end of file
+const userFirstName = document.querySelector('.user-info__greeting');
+const userInfoDisplay = document.querySelector('.user-info__basic-display');
+
+const hydrationCurrentCount = document.querySelector('.hydration__current');
+const hydrationDate = document.querySelector('.hydration__date');
+const hydrationWeek = document.querySelector('.hydration__week');
+
+const sleepCurrentHours = document.querySelector('.sleep__current-hours');
+const sleepCurrentQuality = document.querySelector('.sleep__current-quality');
+const sleepWeekHours = document.querySelector('.sleep__week-hours');
+const sleepWeekQuality = document.querySelector('.sleep__week-quality');
+const sleepAveragesHours = document.querySelector('.sleep__averages-hours');
+const sleepAverageQuality = document.querySelector('.sleep__averages-quality');
+
+const activityCurrentSteps = document.querySelector('.activity__current-steps');
+const activityCurrentMinutes = document.querySelector('.activity__current-minutes');
+const activityCurrentMiles = document.querySelector('.activity__current-miles');
+const activityAverageSteps = document.querySelector('.activity__current-average-steps')
+const activityAverageMinutes = document.querySelector('.activity__current-average-minutes')
+const activityAverageMiles = document.querySelector('.activity__current-average-miles')
+const activityWeekSteps = document.querySelector('.activity__week-steps');
+const activityWeekStairs = document.querySelector('.activity__week-stairs');
+const activityWeekMinutes = document.querySelector('.activity__week-minutes');
+
+
+let allUsers = userData.map(userInfo => {
+  const user = new User(userInfo);
+  return user;
+});
+
+let userRepo = new UserRepository(allUsers);
+
+let today;
+let currentUser;
+let activityInfo;
+let hydrationInfo;
+let sleepInfo
+
+const pageLoad = () => {
+  currentUser = userRepo.userData[47];
+  userFirstName.innerText = `Hello! ${currentUser.returnUserFirstName()}!`;
+  showUserInfo();
+  showActivityInfo();
+  showHydrationInfo();
+  showSleepInfo();
+};
+
+const showUserInfo = () => {
+  const userFriends = currentUser.friends.map(friend => {
+    const userFriend = userRepo.findUser(friend)
+    return userFriend.name
+  })
+  userInfoDisplay.innerHTML = `<li class="user-info__stride-length">Stride: ${currentUser.strideLength}</li>
+  <li class="user-info__friends"> Friends
+    <ul>
+      <li class="friend1">${userFriends[0]}</li>
+      <li class="friend2">${userFriends[1]}</li>
+      <li class="friend3">${userFriends[2]}</li>
+    </ul>
+  </li>
```

Comment:
let's not forget to come back and refactor so that this only populates the DOM for the number of friends an individual user has (rather than displaying `undefined` for `userFriends[2]` if a user only has 2 friends listed in their data